util/src/main/java/io/kubernetes/client/util/ListAccessor.java

```diff
@@ -0,0 +1,32 @@
+package io.kubernetes.client.util;
+
+import io.kubernetes.client.openapi.models.V1ListMeta;
+import io.kubernetes.client.util.exception.ObjectMetaReflectException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+
+/** ListAccessor is a set of utils which accesses common fields from list objects dynamically */
+public class ListAccessor {
+
+  public static final String METHOD_NAME_GET_ITEMS = "getItems";
```

Comment:
Make these private?

Reply:
Given that these strings are only used once, are constants needed?

Reply:
updated to private

Reply:
separate hardcodes as const is aesthetically pleasing, they don't really make a difference tho